src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -337,7 +338,51 @@ def build_resolve_modal_payload(self, callback_id):
             "callback_id": callback_id,
         }
 
-    def build_archive_modal_payload(self, callback_id):
+    def build_resolve_modal_payload(self, callback_id) -> View:
+        formatted_resolve_options = []
+        for text, value in RESOLVE_OPTIONS.items():
+            formatted_resolve_options.append(
+                {
+                    "text": {
+                        "type": "plain_text",
+                        "text": text,
+                        "emoji": True,
+                    },
+                    "value": value,
+                }
+            )
+
+        return View(
+            **{
```

Comment:
Rather than format this with `**`, could we do
```python
View(
    type="modal",
    title=...
)
```
?

If for some reason this is preferable, we can cut down on indentation by extracting the dict such as
```python
kwargs = {
    "type": "modal",
    ...
}
return View(**kwargs)
```